.github/workflows/lint-all.yaml

```diff
@@ -44,3 +44,11 @@ jobs:
       - uses: bufbuild/buf-lint-action@044d13acb1f155179c606aaa2e53aea304d22058
         with:
           input: service
+  
+  pre-commit:
```

Comment:
Can we move this to checks.yaml and delete lint-all?

Reply:
Resolved by deleting this bit of code 👍